expyfun/io/__init__.py

```diff
@@ -1,4 +1,15 @@
 # -*- coding: utf-8 -*-
 from ._wav import read_wav, write_wav
-from ._hdf5 import read_hdf5, write_hdf5
+from .._externals._h5io import (read_hdf5 as _read_hdf5,
+                                write_hdf5 as _write_hdf5)
 from ._parse import read_tab
+
+
+def read_hdf5(fname):
+    """..."""
```

Comment:
One last thing -- that `...` in my code snippet was meant to indicate a placeholder for text to be filled in by you, you can probably keep whatever was in the original `_hdf5.py` file from this repo for `read_hdf5` and `write_hdf5`. I probably should have been clearer on that.